catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -110,7 +113,7 @@ public MetacardImpl(MetacardType type) {
          * serialized object is maintained. For instance, if a null check is added in the
          * constructor, the same check should be added in the readObject() method.
          */
-        map = new HashMap<String, Attribute>();
+        map = new ConcurrentHashMap<>();
```

Comment:
I'm not clear how this is helping to solve the underlying problem. All this concurrent map is going to do is prevent multiple threads from colliding with each other as they update the internals of this metacard, but it won't help with the case where the metacard is being modified after being added to the `CBP`.